css-view-transitions-1/Overview.bs

```diff
@@ -1243,19 +1243,17 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 			1. If any [=flat tree=] ancestor of this |element| [=skips its contents=], then [=continue=].
 
+			1. If |element| is not |element|'s [=tree/root=] and |element| allows [=fragmentation=], then [=continue=].
```

Comment:
```suggestion
			1. If |element| has more than one [=box-fragment=], then [=continue=].
```
root doesn't need to be special cased, we ignore it if its fragmented just the same as other elements. And the resolution was to do this if the box is fragmented during the transition as opposed to whether it can be fragmented.